### ISSUE TITLE
G533 headset wireless not being detected

### ISSUE THREAD

**Issue by andyvorld** [andyvorld/LGSTrayBattery#40]
thanks, now not crashing anymore! 

correctly detected:
Mx Vertical mouse
K360 Keyboard
K400 plus Keyboard

but NOT detected: 
G533 headset wireless

what can i do to make it work even with the headset?

_Originally posted by @wergio in https://github.com/andyvorld/LGSTrayBattery/issues/37#issuecomment-1230644127_

**Comment by andyvorld** [comment]
Did the G533 get detected in previous versions?

**Comment by wergio** [comment]
no, the software was crashing (i think because of G533). there is a log or other data I can provide?

**Comment by andyvorld** [comment]
I think its because that the G533 acts differently from normal peripherals, as seen in https://github.com/pwr-Solaar/Solaar/issues/641

Try replacing hidpp_mon.exe with this custom build, it's not tested very well, so it may not work. If possible try it with just the headset connected.
[x64-release.zip](https://github.com/andyvorld/LGSTrayBattery/files/9460959/x64-release.zip)

**Comment by andyvorld** [comment]
I have merged in the changes from the custom build into the main build in [v2.1.1](https://github.com/andyvorld/LGSTrayBattery/releases/tag/v2.1.1). Let me know if that works.
